tiledb/sm/serialization/query.cc

```diff
@@ -1297,22 +1308,29 @@ Status query_to_capnp(
           buff.original_buffer_size_);
     } else if (buff.buffer_ != nullptr && buff.buffer_size_ != nullptr) {
       // Fixed-length buffer
-      total_fixed_len_bytes += *buff.buffer_size_;
-      attr_buffer_builder.setFixedLenBufferSizeInBytes(*buff.buffer_size_);
+
```

Comment:
NIT: consider removing extra line breaks in the next few dozen lines of this review.